pkg/util/azureclient/environments.go

```diff
@@ -58,6 +62,8 @@ var (
 		AppLensScope:             "https://microsoft.onmicrosoft.com/runtimehost",
 		AppLensTenantID:          "cab8a31a-1906-4287-a0d8-4eef66b95f6e",
 		Cloud:                    cloud.AzureGovernment,
+		PkiIssuerUrlTemplate:     "",
```

Comment:
can you add comment on why they are empty

Reply:
Done